Updated Logos

yatil opened the pull request:
I hope this updates the new logo, I wasn’t able to compile/test it locally. I only got a white page using `grunt serve`. (I hope this is the right branch to merge into, it was very hard to find where the development was going on.)

Rel. #305

rvantonisse commented:
Thanks yatil, yes correct branch, but might even merge as hotfix into master since this is a brand update. Checking this out.

rvantonisse commented:
@yatil you might need to double install bower components. simply run `npm install` twice for ease.

You can then serve with `npm start` or build with `npm run build`

rvantonisse commented on app/views/navigation.html:
```diff
@@ -3,16 +3,15 @@
       <div class="nav navbar-nav navbar-right">
         <span class="w3c"><a href="http://w3.org/">
-          <img alt="W3C" width="85" height="50" src="images/w3c.png">
+          <img alt="W3C" width="110" height="75" src="images/w3c.png">
```
Undo height changes please, this breaks current design, or is there a reason it needs to be this size?
(And by all means keep the right aspect ratio for the logo)

rvantonisse commented on app/views/navigation.html:
```diff
@@ -3,16 +3,15 @@
         </a></span>
         <span class="wai"><a href="http://w3.org/WAI/">
-          <img alt="Web Accessibility Initiative" width="194" height="48"
+          <img alt="Web Accessibility Initiative" width="180" height="75"
```
Same here please undo height changes.

rvantonisse commented:
Possibly totally redundant but these are the logos changed:
![W3C](https://github.com/w3c/wcag-em-report-tool/tree/updated-logo/app/images/w3c.png)

![WAI](https://github.com/w3c/wcag-em-report-tool/tree/updated-logo/app/images/wai.png)

yatil commented:
@slhenry I updated the logo and changed some of the colors to better work with the new design. No time  for a comprehensive update, but I think it’s also not high priority at this point.

Screen shot that shows the changes:

![Screen Shot(https://user-images.githubusercontent.com/58862/70515566-d80a6d00-1b35-11ea-950a-7b4e35f6ae73.png)

@rvantonisse Feel free to merge however you feel it should be merged once @slhenry has approved it. I don’t feel strongly. (Feel free to squash the merge, no need to keep the pngs in history :-D)

slhenry commented:
+1
